examples/transforms/global_vs_local_translation.rs

```diff
@@ -116,11 +134,15 @@ fn setup(
 
     // Add text to explain inputs and what is happening.
     commands.spawn(TextBundle::from_section(
-        "Press the arrow keys to move the cubes. Toggle movement for yellow (1), red (2) and green (3) cubes via number keys.
-
-Notice how the green cube will translate further in respect to the yellow in contrast to the red cube.
-This is due to the use of its LocalTransform that is relative to the yellow cubes transform instead of the GlobalTransform as in the case of the red cube.
-The red cube is moved through its GlobalTransform and thus is unaffected by the yellows transform.",
+        "Press the arrow keys to move the cubes. \
+        Toggle movement for yellow (1), red (2) and green (3) cubes via number keys.\n\n\
+        Notice how the green cube will translate further in respect to the \
+        yellow in contrast to the red cube.\n\
+        This is due to the use of its LocalTransform that is relative to the \
+        yellow cubes transform instead of the GlobalTransform as in the case of the red cube.\n\
+        The red cube is moved through its GlobalTransform and thus is \
+        unaffected by the yellows transform.\n\
+        You can toggle the parent relationship between the green and yellow cubes using (4)",
```

Comment:
If I press (4) to swap the parents while yellow is paused, the red cube teleports. Is that supposed to happen?

Reply:
This is really weird. Took me a while to figure it out, but looks like the culprit is the `propagate_transforms` systems. It updates the `GlobalTransform` (therefore resets it to the yellow `GlobalTransform` times red's `Transform`) of the red cube whenever the `Children` component of the yellow cube is updated. Which is of course very surprising.

Not sure how to deal with this. :grimacing: I'll open a PR to remove the `translation_mut` method on `GlobalTransform` though.

Reply:
Do you know if the bug existed before this PR or not?

Reply:
It happens now because reparenting triggers the recomputation and overrides the user-set `GlobalTransform`. Before, there was no option to unparent/reparent in the example, it relied on the fact that the propagation system doesn't trigger for yellow. (It doesn't trigger for yellow because the example update yellow's `GlobalTransform` instead of its `Transform`)